jekyll/_cci2/contexts.md

```diff
@@ -237,121 +237,110 @@ to set up your CircleCI command line interface._
-  `circleci context show <vcs-type> <org-name> <context-name>`
-3. Delete the environment variable by executing this command:\
-  `circleci context remove-secret <vcs-type> <org-name> <context-name> <secret name>`
-
-##### Using CircleCI’s API
-{:.no_toc}
+To delete an environment variable using the CLI, perform the following steps:
 
-To delete an environment variable using our API, perform the following steps:
+1. If you have not already done so, find the context name that contains the 
+   environment variable you wish to delete. Execute this command in the CLI:
+   `circleci context list <vcs-type> <org-name>`
 
-1. Call the [Delete Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext) endpoint in one of our supported languages. For this request, replace the `context-id` and the `env-var-name` with the ID for the context and the environment variable name that should be updated. 
+2. Confirm the environment variable exists within that context. Execute this 
+   command in the CLI to list all variables under that context:
+   `circleci context show <vcs-type> <org-name> <context-name>`
 
-    To see request samples in all supported languages, visit our [api docs on creating an environment variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext).
+3. Delete the environment variable by executing this command:
+   `circleci context remove-secret <vcs-type> <org-name> <context-name> <env-var-name>`
 
-2.  If the request is successful, the response will include a message object
-    confirming the variable was successfully deleted. An unsuccessful response
-    will include a clear error message explaining the error reported.
+##### Using CircleCI’s API
+{:.no_toc}
 
+To delete an environment variable using the API, call the [Delete Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext) endpoint.
 
-#### Rotating Context Environment Variables
+For this request, replace the `context-id` and the `env-var-name` with the ID 
+   for the context and the environment variable name that should be updated.
 
+#### Rotating Environment Variables
 
-Rotation occurs when an old API token is replaced with a new token. 
+Rotation refers to the process of updating a secret's value without deleting it 
+or changing its name.
 
 Because environment variables can be shared, passed around between employees and
 teams, and exposed inadvertently, it is always good practice to periodically
 rotate secrets. Many organizations automate this process, running a script when
-an employee leaves the company or when a secret has been considered leaked.
+an employee leaves the company or when a secret may have been compromised.
 
 Context environment variables can be rotated using CircleCI’s CLI, or by
-directly accessing our API.
+directly accessing the API.
```

Comment:
Just making "our" vs. "the" consistent (IME we use "the" more often in product docs)